bluemira/equilibria/coils/_tools.py

```diff
@@ -83,10 +83,22 @@ def make_mutual_inductance_matrix(coilset: CoilSet) -> np.ndarray:
 
 def _get_symmetric_coils(
     coilset: CoilSet,
+    rtol: float | None = None,
```

Comment:
Probably this is fine and deleting the or test below and feeding that up through
```suggestion
    rtol: float = 1e-5
```